src/Silex/Provider/SwiftmailerServiceProvider.php

```diff
@@ -91,26 +91,30 @@ public function register(Container $app)
 
             $plugins = $app['swiftmailer.plugins'];
 
+            $keys = $plugins->keys();
+            sort($keys);
```

Comment:
Keeping the registration order (which is what ``$plugins->keys()`` does) looks better indeed